### PR TITLE
 ramips: various Wavlink WL-WN575A3 fixes 

### PR DESCRIPTION
**Pull request by tvc**
Fixed some discrepancies and updated board detection for the Wavlink WL-WN575A3.

Signed-off-by: Thomas Vincent-Cross <me@tvc.id.au>

### CODE REVIEW

File: target/linux/ramips/image/mt76x8.mk [hunk 1]
```diff
@@ -352,6 +352,14 @@ define Device/wavlink_wl-wn570ha1
 endef
 TARGET_DEVICES += wavlink_wl-wn570ha1
 
+define Device/wavlink_wl-wn575a3
```
Comment (chunkeey) [inline target/linux/ramips/image/mt76x8.mk]: Please include a SUPPORTED_DEVICES += wl-wn575a3 so older installations can upgrade without a forced sysupgrade.

Comment (tvc) [reply target/linux/ramips/image/mt76x8.mk]: Oh cool, I didn't realise that was an option. Fixed.

**Review by chunkeey — APPROVED** [review]

**Comment by chunkeey** [comment]
Merged. Thanks.

**Comment by adrianschmutzler** [comment]
Hi, I'm currently reviewing the MAC address assignment on ramips target.
For your device, you are currently using MAC address from <&factory 0x28> and then just calculate WAN address by adding +1 to that.
However, there might be another MAC address stored on flash for your device in <&factory 0x2e>.

Can you check that? If yes, I would also be interested in which of the two addresses should be "lan" and which "wan" then ...

**Pull request closed without merging**